pandas/io/parquet.py

```diff
@@ -404,10 +416,12 @@ def to_parquet(
         return None
 
 
+@doc(storage_options=generic._shared_docs["storage_options"])
 def read_parquet(
     path,
     engine: str = "auto",
     columns=None,
+    storage_options: StorageOptions = None,
```

Comment:
need Optional here

Reply:
`StorageOptions` is already defined with an `Optional`: `StorageOptions = Optional[Dict[str, Any]]`

Reply:
ahh ok